### ISSUE TITLE
Supported/tested Linux distros and versions

### ISSUE THREAD

**Issue by legooolas** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#3590]
Which Linux distros are the packages (both RPM and Deb format) tested and supported on?
A table showing which distros and versions it's tested and/or expected to work on would be very useful, if possible!
I ask because the rpm version doesn't install on CentOS 7.7 as it requires a newer libstdc++ than is provided, so I am wondering if it is only supported on CentOS 8.x as the newest release?
(CentOS 7 EOL is June 30, 2024, so it will be around a long while yet!)

  Many thanks :)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6457d204-d229-9154-28fd-ca9bbb4bddcd
* Version Independent ID: f5d82846-41cb-3548-6997-ce23bd8f89c6
* Content: [Get clients for Microsoft Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/get-clients#feedback)
* Content Source: [Teams/get-clients.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/get-clients.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

**Comment by legooolas** [comment]
Even just which glibc and libstdc++ library versions are required for this to work would be useful, since that would allow users to check whether they meet the minimum requirements or need to upgrade or find some sort of workaround to get the client to work  :)

**Comment by scanum** [comment]
@legooolas Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by TokyoScarab** [comment]
@legooolas  Thank you for raising this issue! I'll be looking into it!

**Comment by TokyoScarab** [comment]
@LolaJacobsen Would you happen to have any information on this or who to contact who might be able to provide some information that could be used to update the documentation?

**Comment by scanum** [comment]
Hi @LolaJacobsen Could you please help on this issue? Thanks!

**Comment by kenwith** [comment]
I added in the Linux content. I can take this one.

**Comment by kenwith** [comment]
I just realized this issue is very old and the documentation has already been updated. There is also an Answers thread here that might be helpful for anyone looking for the discussion on when this was resolved: https://docs.microsoft.com/answers/comments/6721/view.html

**Comment by legooolas** [comment]
The documentation still doesn't say which distros and versions it's tested/supported on.
The answers page linked to says that CentOS 7 is now supported, but the original question stands of "Which Linux distros are the packages (both RPM and Deb format) tested and supported on?"

**Comment by kenwith** [comment]
@legooolas thanks for this feedback. I will reach out to the product team and get this information and get it added to the article.

**Comment by scanum** [comment]
@kenwith Hi Ken! Any update on this issue? Thanks!

**Comment by kenwith** [comment]
Yes, I have contacted the product team and they are working out the best way to tackle a supported list. I will keep after this and get the documentation updated as soon as possible.

**Comment by hadenodom** [comment]
I have installed teams on two computers:  One laptop running Ubuntu and one desktop running Arch Linux.  Both are on the latest approved stable version, and both are running the latest version of Teams, straight from the MS website as of today.

On the Ubuntu machine, it's running fine and I've run into no issues.  On the desktop running Arch, though, it shows a "We're still developing Teams for Linux" page when I log in.  Could this be caused by Arch not being officially supported (and your software doing some sort of check to see if it's on an approved distro?)  It's pretty annoying to have software that you *know* works, but it won't let you use it because... well, because there's a chance it might not work.  I don't get to test it for myself, apparently.

**Comment by kenwith** [comment]
I just wanted to send a quick update that the product team is in the process of releasing a Snap for Teams. I will be updating the documentation here with links and also on the requirements page once it is released. I also sent the feedback about the issue @hadenodom surfaced regarding the login page and will update this issue and the docs based on info I get.

**Comment by scanum** [comment]
@kenwith Hi! Wondering if there are any updates on this issue? Thanks.

**Comment by scanum** [comment]
@kenwith Hi! Wondering if there are any updates on this issue? Thanks.

**Comment by scanum** [comment]
@kenwith Hi! Wondering if there are any updates on this issue? Thanks.

**Comment by kenwith** [comment]
@scanum - handing this over to Serdar who will be assigning someone to take over the Linux documentation. The latest information I have is that the product team is in the process of release a Snap and the documentation will be updated with that information as soon as it is available.
